src/components/JdsSimpleTable/SimpleTable.scss

```diff
@@ -0,0 +1,34 @@
+@use "~/styles/scss/variables" as V;
+
+.jds-simple-table {
+  width: 100%;
+  border-spacing: 0 !important;
+  border: none;
+  
+  * {
+    font-family: inherit;
+    font-size: 14px;
+    line-height: 23px;
+  }
+
+  tr {
+    background-color: V.$white;
+    text-align: left;
+
+    th {
+      background-color: V.$green-700;
+      padding: 6px 8px;
+      color: V.$white;
+      font-weight: 700;
+    }
+  
+    td {
+      padding: 6px 8px;
+      border-top: 1px solid V.$gray-200;
+    }
+
+    &:nth-child(even) {
+      background-color: V.$gray-50;
+    }
+  }
+}
```

Comment:
Low Priority
It's better to be given a new line

Reply:
thankyou, will fix it asap

Reply:
Done [`da0ee94`](https://github.com/jabardigitalservice/jds-design-system-vue-lib/pull/42/commits/da0ee94725188f21503b0f33f44ee08915c087bd)